kernel/kernel-api/src/main/java/io/delta/kernel/internal/DeltaLogActionUtils.java

```diff
@@ -261,22 +266,38 @@ private static CloseableIterator<FileStatus> listLogDir(
    */
   private static List<FileStatus> listCommitFiles(
       Engine engine, Path tablePath, long startVersion, long endVersion) {
-
+    
+    // TODO update to support coordinated commits; suggested to load the Snapshot at endVersion
+    //  and get the backfilled/unbackfilled commits from the LogSegment to combine with commit files
+    //  listed from [startVersion, LogSegment.checkpointVersion]
```

Comment:
I've also added this note since we had a discussion on this and I think is relevant context for future implementation. I've already added updating getChanges to the coordinated commits project plan.